test/blackbox-tests/test-cases/dune

```diff
@@ -1,5 +1,8 @@
 (env
  (_
+  (env-vars
+   (DUNE_CONFIG__BACKGROUND_SANDBOXES disabled)
+   (DUNE_CONFIG__BACKGROUND_DIGESTS disabled))
```

Comment:
Why this diff?

Reply:
Are you saying that before this change builds we're somehow fully deterministic? That seems unlikely.

Reply:
I guess they were non-deterministic but perhaps in ways unobservable in current tests.

Reply:
Yes, they were deterministic enough for the test output to be reliable. With these toggles enabled, it's no longer the case.